modules/API/pages/built-in-endpoints.adoc

```diff
@@ -3384,6 +3384,168 @@ Response::
 | String ID of the query.
 |===
 
+
+=== Get information about a template query
+
+`GET /gsql/library?<function name>=pkg1.pkg2.<template name>`
+
+This endpoint returns the query definition, query name, and query parameters for a template query.
+
+==== Sample request
+[tabs]
+====
+Request::
++
+--
+[source.wrap,bash]
+----
+curl -X GET --user tigergraph:tigergraph --url 'http://127.0.0.1:8123/gsql/library?functionName=gds.community.louvain'
```

Comment:
please do not use 8123 to document for the port is used internally.
it is fine to use http://{host}:14240/gsqlserver/gsql/library?xxx=xxx.
e.g. https://docs.tigergraph.com/gsql-ref/current/querying/query-optimizer/stats-api#_compute_cardinality_statistics